src/js/apps/patients/patients-main_app.js

```diff
@@ -39,6 +39,10 @@ export default RouterApp.extend({
       action: 'showPatient',
       route: 'patient/:id/action/:id',
     },
+    'patient:action:archive': {
+      action: 'showPatient',
+      route: 'patient/archive/:id/action/:id',
```

Comment:
so needing to wait on the context of the action id to understand if we are on an archive page or a patient dashboard was making this complicated..  granted this is going to spill over into the worklists as well, but I think swapping the url is worth the simplicity and speeding up loading the dashboards.

Reply:
@paulfalgout does backend need to create a new endpoint or provide additional data somehow?

Reply:
no essentially a patient dashboard url is `/patient/dashboard/id` and archive is `/patient/archive/id` but for either page an action sidebar url (added later) is `patient/id/action/id` so if you hard refresh it has to request the action prior to knowing if we should be showing the dashboard or the archive.. this just makes the sidebar on an archive more explicit so we don't need to do requests in sequence.